tslib/src/functions/check-marker/check-marker.v1.ts

```diff
@@ -33,16 +33,16 @@ export class CheckMarkerV1 {
     const markingData = await this.validateData(functionBindings, validatedCheck, logger)
     functionBindings.checkNotificationQueue = []
     if (markingData === undefined) {
-      const notification: ICheckNotificationMessage = {
-        checkCode: validatedCheck.RowKey,
-        notificationType: CheckNotificationType.checkInvalid,
-        version: 1
-      }
-      functionBindings.checkNotificationQueue.push(notification)
+      this.notifyProcessingFailure(validatedCheck, functionBindings)
+      return
```

Comment:
isn't the return redundant here?

Reply:
no, because then the next block will run and notify the teacher of completion, which is not wanted